app/models/certificate.server.ts

```diff
@@ -0,0 +1,64 @@
+import type { Certificate } from '@prisma/client';
+
+import { prisma } from '~/db.server';
+
+export type { Certificate } from '@prisma/client';
+
+export async function getCertificateByUsername(username: Certificate['username']) {
+  return prisma.certificate.findUnique({ where: { username } });
+}
+
+export async function createCertificate(
+  username: Certificate['username'],
+  subject: Certificate['subject'],
+  certificate: Certificate['certificate'],
+  orderUrl: Certificate['orderUrl'],
+  privateKey: Certificate['privateKey']
+) {
+  // Set expiration date 90 days from now
+  let validTo = new Date();
```

Comment:
Minor thing, but I think it is better to use `let` only in case if the variable is mutable, and `const` in all other cases

Reply:
Oh, good catch. I forgot that it's an object so you can do that.